modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebarTreeView.tsx

```diff
@@ -371,6 +371,7 @@ export default function LeftSidebarTreeView({
 										'lfr-objects__model-builder-left-sidebar-item': selected,
 										'lfr-objects__model-builder-left-sidebar-item-linked': linked,
 									})}
+									onClick={(event) => event.preventDefault()}
```

Comment:
hey @Artartur! We ended up losing functionality with this change. We can no longer select a definition node by treeviewItem.

![prevent-default](https://github.com/liferay-objects/liferay-portal/assets/61854510/ba38f7cd-69a6-4ad2-90b6-62f297a70644)

 Could you try to improve this fix?